lib/rate-limiter.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const _get = require('lodash.get');
+
+exports.register = (defaultRate, routes) => {
+  const rateTable = {};
+
+  routes.forEach((route) => {
```

Comment:
@robinjoseph08 is saying that reduce makes sense because you are reducing (aggregating) into the `rateTable`.